src/core/instructions.jl

```diff
@@ -200,10 +200,10 @@ Base.getindex(iter::PhiIncomingSet, i) =
     tuple(Value(API.LLVMGetIncomingValue(ref(iter.phi), Cuint(i-1))),
                 BasicBlock(API.LLVMGetIncomingBlock(ref(iter.phi), Cuint(i-1))))
 
-function Base.append!(iter::PhiIncomingSet, args::Vector{Tuple{Value, BasicBlock}})
+function Base.append!(iter::PhiIncomingSet, args::Vector{Tuple{V, BasicBlock}} where V <: Value)
     vals, blocks = zip(args...)
-    API.LLVMAddIncoming(ref(iter.phi), ref.(vals),
-                        ref.(blocks), length(args))
+    API.LLVMAddIncoming(ref(iter.phi), collect(ref.(vals)),
```

Comment:
Why the `collect`s?

Reply:
It is in the top post. These are otherwise tuples and we can't get a pointer to them.